js/modules/k6/execution/execution.go

```diff
@@ -227,6 +241,47 @@ func newInfoObj(rt *goja.Runtime, props map[string]func() interface{}) (*goja.Ob
 	return o, nil
 }
 
+// optionsAsObject maps the lib.Options struct that contains the consolidated
+// and deriverd options configuration in a goja.Object.
+func optionsAsObject(rt *goja.Runtime, options lib.Options) (*goja.Object, error) {
+	// TODO: refactor removing the json part
+	// and replace it with a static data mapping,
+	// potentially using a generator.
+	b, err := json.Marshal(options)
+	if err != nil {
+		return nil, fmt.Errorf("failed to encode the lib.Options as json: %w", err)
```

Comment:
I kind of prefer the `json.Marshal` as it guarantees that we decouple the actual options and what we return.

Reply:
I removed the TODO